### PR TITLE
Spelling: Translations contributed to, "No project assoc"

### PR DESCRIPTION
**Pull request by comradekingu**
Solves https://hosted.weblate.org/translate/weblate/application/en/?checksum=2002e6a7d5bcf841

## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [ ] I have described the changes in the commit messages.

### CODE REVIEW

File: weblate/templates/accounts/user.html [hunk 1]
```diff
@@ -138,7 +138,7 @@ <h3>{{ page_user }}</h3>
 </div>
 
 <ul class="nav nav-pills nav-center">
-<li class="active"><a data-toggle="tab" href="#contributed">{% trans "Translations with contribution" %}</a></li>
+<li class="active"><a data-toggle="tab" href="#contributed">{% trans "Translations contributed to" %}</a></li>
```
Comment (comradekingu) [inline weblate/templates/accounts/user.html:141]:
```suggestion
<li class="active"><a data-toggle="tab" href="#contributed">{% trans "Contributed to" %}</a></li>
```
Possible

File: weblate/templates/accounts/user.html [hunk 2]
```diff
@@ -182,7 +182,7 @@ <h3>{{ page_user }}</h3>
 
 <div class="tab-pane" id="changes">
     {% include "last-changes-content.html" %}
-    <a class="btn btn-link" href="{% url 'changes' %}?{{ last_changes_url }}">{% trans "Browse all changes for user" %}</a>
+    <a class="btn btn-link" href="{% url 'changes' %}?{{ last_changes_url }}">{% trans "Browse all changes by this user" %}</a>
```
Comment (comradekingu) [inline weblate/templates/accounts/user.html:185]:
```suggestion
    <a class="btn btn-link" href="{% url 'changes' %}?{{ last_changes_url }}">{% trans "Browse all changes made by this user" %}</a>
```
If it fits

File: weblate/templates/accounts/user.html [hunk 3]
```diff
@@ -182,7 +182,7 @@ <h3>{{ page_user }}</h3>
 
 <div class="tab-pane" id="changes">
     {% include "last-changes-content.html" %}
-    <a class="btn btn-link" href="{% url 'changes' %}?{{ last_changes_url }}">{% trans "Browse all changes for user" %}</a>
+    <a class="btn btn-link" href="{% url 'changes' %}?{{ last_changes_url }}">{% trans "Browse all changes made by this user" %}</a>
```
Comment (nijel) [inline weblate/templates/accounts/user.html:185]:
```suggestion
    <a class="btn btn-primary" href="{% url 'changes' %}?{{ last_changes_url }}">{% trans "Browse all changes made by this user" %}</a>
```

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**